app/assets/stylesheets/revised/pages/landing/_home.scss

```diff
@@ -12,18 +12,177 @@
+}
+
+.root-landing-banner-subheader {
+  width: 100%;
+  margin: 0;
+
+  .subheader-tile-link {
+    color: #fff;
+    text-decoration: none;
+  }
+
+  .subheader-tile:first-of-kind {
+    padding-right: 5px;
+  }
+
+  .subheader-tile:last-of-kind {
+    padding-left: 5px;
+  }
+
+  .subheader-tile {
+    margin: 10px auto;
+    min-width: 275px;
+    padding-left: 5px;
+    padding-right: 5px;
+  }
+
+  .subheader-image {
+    height: 275px;
+  }
+
+  .subheader-image-caption {
+    background-color: black;
+    color: #fff;
+    font-weight: bold;
+    padding: 10px 15px;
+    text-decoration: none;
+  }
+}
+
+.center-navbar-signup-link-container {
+  display: inline-block;
+  margin: 20px auto 10px auto;
+  position: absolute;
+  text-align: center;
+  width: 75%;
+}
+
+.center-navbar-signup-link {
+	background-color: black;
+	border-radius: 4px;
+	border: 1px solid #3498DB;
+	width: 85px;
+  box-sizing: border-box;
+  color: #fff;
+  cursor: pointer;
+  display: inline;
+  font-size: 12px;
+  height: 37px;
+  line-height: 18px;
+  padding: 8px 15px;
+  text-align: center;
+  text-decoration: none;
+
+  &:hover {
+    background: rgba(white, 0.1);
+    text-decoration: none;
+    color: #fff;
+  }
+
+  &.active,
+  &:active {
+    background: rgba(white, 0.1);
+    text-decoration: none;
+    color: #fff;
+  }
+}
+
+@media screen and (min-width: 991px) {
+  .center-navbar-signup-link-container {
+    display: none;
+  }
+}
+
+@media screen and (max-width: 1099px) {
+  .root-landing-banner-subheader {
+    .subheader-tile {
+      margin: 10px auto;
+    }
+  }
+}
+
+@media screen and (max-width: 500px) {
+  .root-landing-banner-subheader {
+    .subheader-tile {
+      margin: 0 auto;
+      min-width: 100%;
+    }
+  }
+
+  .subheader-image {
+    display: none;
+  }
+
+  .subheader-image-caption {
+    margin: 5px auto;
+  }
+}
+
+.bg-image-bike-hero {
+  background-image: image-url("landing_pages/header_bg_root-full.jpg");
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.bg-image-bike-shops {
+  background-image: image-url("landing_pages/bike-shop.jpg");
   background-size: cover;
-  background-position: center bottom;
+  background-position: top;
   background-repeat: no-repeat;
 }
 
+.bg-image-bike-cities {
+  background-image: image-url("landing_pages/cities.jpg");
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.bg-image-bike-schools {
+  background-image: image-url("landing_pages/schools.jpg");
+  background-size: cover;
+  background-position: top;
+  background-repeat: no-repeat;
+}
+
+.bg-image-bike-groups {
+  background-image: image-url("landing_pages/groups.jpg");
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+@include media-breakpoint-up(lg) {
+  .bg-image-bike-hero {
+    background-position: bottom;
+  }
+
+  .bg-image-bike-shops {
+    background-position: center top;
```

Comment:
Since these are all different, I'm guessing you checked where the background looked best at different screen sizes, which is above and beyond!